src/Module.Server/Rewards/CrpgRewardClient.cs

```diff
@@ -60,6 +60,16 @@ private void HandleRewardUser(CrpgRewardUser message)
                     color));
         }
 
+        if (message.TotalCompensation != 0)
+        {
+            (Color color, string verb, string teamhitMessage) = message.TotalCompensation > 0
+                ? (new Color(65, 105, 225), "Gained", "being teamhit")
+                : (new Color(0.74f, 0.28f, 0.01f), "Lost", "teamhitting others");
+            InformationManager.DisplayMessage(
+                new InformationMessage($"{verb} {message.TotalCompensation} gold as compensation for {teamhitMessage}.",
+                    color));
+        }
+

```

Comment:
I had this version previously, but i like the current version more, because you can instantly see (via color code) if you needed to pay or got paid.
![grafik](https://github.com/verdie-g/crpg/assets/35863824/42e5584b-eda5-4e93-9427-9c3320fc020b)

If i were to put it in the reward message, it would lose the instant recognition of either penalty or reward. You would need to read first. But ultimatly i'm fine with both, your call what is better.
![grafik](https://github.com/verdie-g/crpg/assets/35863824/3f2ca221-1d7c-493e-8720-98ef7b6b6af7)

Reply:
@verdie-g  i asked for it to not be merged as i'd like one message to answer the question "am i profittable with current loadout" and the second message to answer "am i a teamkiller?"

Reply:
I still would prefer it with a separate message, because you can instantly see (via color code) if you needed to pay or got paid.
Another idea was to color code each argument (reward, upkeep, compensation) but Taleworlds only allows for one Color for the whole Message.